rllib/algorithms/algorithm_config.py

```diff
@@ -2195,21 +2202,18 @@ def debugging(
 
     def fault_tolerance(
         self,
-        ignore_worker_failures: Optional[bool] = NotProvided,
         recreate_failed_workers: Optional[bool] = NotProvided,
         max_num_worker_restarts: Optional[int] = NotProvided,
         delay_between_worker_restarts_s: Optional[float] = NotProvided,
         restart_failed_sub_environments: Optional[bool] = NotProvided,
         num_consecutive_worker_failures_tolerance: Optional[int] = NotProvided,
         worker_health_probe_timeout_s: int = NotProvided,
         worker_restore_timeout_s: int = NotProvided,
+        ignore_worker_failures=DEPRECATED_VALUE,
```

Comment:
I think we don't need to include it here anymore :). The whole API is new.

Reply:
huh, that's very true :)